README.rst

```diff
@@ -9,16 +9,16 @@ Contributing
 
 This project only requires that:
 
-#. you have `tox`_ installed
-#. you have Python 3 installed locally
+#. you have `sphinx`_ installed (`pip install sphinx`)
 
 To contribute to this repository, fork the project and make your changes.
-Before submitting a pull request, please check that your changes render as 
+Before submitting a pull request, please check that your changes render as
 expected by running::
 
-    $ tox -e docs,serve
+    $ make html
```

Comment:
This is the bit that is not negotiable for me. I'd like to give people on Windows an easy way to contribute. We do not presently have a great number of people but we're removing the simplicity of one command with needing to know how to work around the first and do the second in a platform specific way.

If you prefer to work this way, that's great. I don't think we should be forcing this on others (or implicitly asking them to install sphinx globally). (Further, sphinx's generated `Makefile` apparently doesn't play very well with virtualenvs.)